Sources/Sentry/SentryDispatchFactory.m

```diff
@@ -0,0 +1,36 @@
+#import "SentryDispatchFactory.h"
+#import "SentryDispatchQueueWrapper.h"
+#import "SentryDispatchSourceWrapper.h"
+
+@implementation SentryDispatchFactory
+
+- (SentryDispatchQueueWrapper *)queueWithName:(const char *)name
+                                   attributes:(dispatch_queue_attr_t)attributes
+{
+    return [[SentryDispatchQueueWrapper alloc] initWithName:name attributes:attributes];
+}
+
+- (SentryDispatchSourceWrapper *)sourceWithInterval:(uint64_t)interval
+                                             leeway:(uint64_t)leeway
+                                              queue:(SentryDispatchQueueWrapper *)queueWrapper
+                                       eventHandler:(void (^)(void))eventHandler
+{
+    return [[SentryDispatchSourceWrapper alloc] initTimerWithInterval:interval
+                                                               leeway:leeway
+                                                                queue:queueWrapper
+                                                         eventHandler:eventHandler];
+}
```

Comment:
m: This function is only used once, and internally in this same class. Can we remove it?